main.go

```diff
@@ -72,9 +95,97 @@ func init() {
 	randSource := mathrand.NewSource(seed)
 	rng = mathrand.New(randSource)
 
+	setDefaultAttributes()
+
 	go sendWorkerMain()
 }
 
+func setDefaultAttributes() {
+	if Options.Attributes == nil {
```

Comment:
Worth mentioning - the user has the power to override those attributes. If the attribute exists in the Options.Attributes it should be respected over default we're populating.

Example:
if the attribute is leaking something (for example PII info for hostname), then the user should be able to override it.

Suggestion - let's merge two maps together and allow Options.Attributes to override your custom map

Reply:
Function is called in init()
Which is called only once when package is imported.
User can override it without any problems. :D